lib/Thruk/Authentication/User.pm

```diff
@@ -140,6 +143,12 @@ sub new {
         $self->{'roles'} = [ grep({ $_ ne 'authorized_for_read_only' } @{$self->{'roles'}}) ];
     }
 
+    # Does this user have all roles?
+    if (@{$self->{'roles'}} == @{$possible_roles}) {
+        # Yes he/she does. Remove read only role
+        $self->{'roles'} = [ grep({ $_ ne 'authorized_for_read_only' } @{$self->{'roles'}}) ];
+    }
+
```

Comment:
i guess we have to add the new roles here to anyone who is a admin (using the logic if someone has both, authorized_for_system_information and authorized_for_configuration_information). Or we do this check later. Anyway, anyone with those two roles must still be able to access broadcasts, reports and BPs.

Reply:
Sounds like a good idea. Done, please see 73fcdbc9444a46aec5f49fe1daa51cd2afee8f6f